src/handlers/register_client.rs

```diff
@@ -61,7 +60,15 @@ pub async fn handler(
     };
 
     if exists {
-        return Err(ClientAlreadyRegistered);
+        state
+        .client_store
+        .update_client(&tenant_id, &body.client_id, Client {
```

Comment:
Isn't there a risk of somebody spoofing the client id and taking over an account?

Reply:
Yes see #57